python/test/test_polars.py

```diff
@@ -548,3 +548,69 @@ def test_replace_binary_path():
     assert str(m_expr((pl.LazyFrame(dict()), pl.all()))) == "len().testing!:noise_plugin()"
 
     del os.environ["OPENDP_LIB_PATH"]
+
+
+def test_pickle_bomb():
+    pl = pytest.importorskip("polars")
+
+    from polars._utils.parse import parse_into_list_of_expressions  # type: ignore[import-not-found]
```

Comment:
```suggestion
    from polars._utils.parse import parse_into_list_of_expressions  # type: ignore[import-not-found]
    from polars._utils.wrap import wrap_expr
```
Do all the imports at the top?

I'm wary of the imports from `_utils`: Definitely wouldn't rely on this outside of tests, but maybe if it helps us write a clean test, it's worth the risk? Any way we could use a higher-level, public interface?

Reply:
To use a higher-level Polars API, I'd have to further reverse-engineer the pickle protocol. I'd rather not spend more time on that, when this is so simple already.